lib/GroupBackend.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace OCA\User_SAML;
+
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use Doctrine\DBAL\FetchMode;
+use OCA\User_SAML\Exceptions\AddUserToGroupException;
+use OCP\Group\Backend\ABackend;
+use OCP\Group\Backend\IAddToGroupBackend;
+use OCP\Group\Backend\ICountUsersBackend;
+use OCP\IDBConnection;
+
+class GroupBackend extends ABackend implements IAddToGroupBackend, ICountUsersBackend {
```

Comment:
Need to specify that this implements IAddToGroupBackend, IRemoveFromGroupBackend and a few other interface

Reply:
also methods need return type hints